testing/newt/APKBUILD

```
@@ -0,0 +1,35 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=newt
+pkgver=1.5.0
+_ver=1_5_0_tag
```

Note entirely sure if it's acceptable, because it's not POSIX complient, but ash does support it, but if you use this, then we don't have to update both `pkgver` and `_ver`.
```suggestion
_ver=${pkgver//./_}_tag
```
There are existing packages that use this syntax.

---

Done! Thank you!